binaries/build.py

```diff
@@ -23,26 +41,58 @@ def build():
     print(f"## In directory: {os.getcwd()} | Executing command: {create_wheel_cmd}")
     ma_build_exit_code = os.system(create_wheel_cmd)
 
+    os.chdir(REPO_ROOT)
+    
+    # Build model archiver wheel
+    os.chdir("workflow-archiver")
```

Comment:
Do you mean the lines to chdir? - they've existed from previous script and I didn't really modify them because the idea seems to me to retain the command to build wheel files, and to simply move around the directories to run the same command.